services/notification-service/app/main.py

```
@@ -0,0 +1,111 @@
+        nc = await nats_lib.connect(
+            settings.nats_url,
+            connect_timeout=settings.nats_connect_timeout_seconds,
+        )
+        set_nats_client(nc)
+        await nc.subscribe(settings.nats_subject_order_completed, cb=_handle_nats_message)
+        logger.info(
+            "nats_subscription_started",
+            nats_url=sanitized_nats_url,
+            subject=settings.nats_subject_order_completed,
+        )
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)